client/shipping-label/index.js

```diff
@@ -68,7 +67,7 @@ export default ( { formData, labelsData, paperSize, storeOptions, paymentMethod
 						retrievalInProgress: false,
 					},
 				},
-				openedPackageId: labelsData ? '' : Object.keys( formData.selected_packages )[ 0 ] || '',
+				openedPackageId: Object.keys( formData.selected_packages )[ 0 ] || '',
```

Comment:
This was needed in the case that all items were in the "saved for later" list and there were no packages. But in the new paradigm, we'll always at least have one package. Speaking of which, I just realized there's a major bug in our UI.... If you want to move everything to individual packages, you won't be able to delete the remaining empty box. I'll get that fixed.